src/rdf4cpp/rdf/ClosedNamespace.hpp

```diff
@@ -23,7 +22,7 @@ class ClosedNamespace : public Namespace {
     ClosedNamespace(std::string_view namespace_iri, Suffixes all_suffixes, NodeStorage &node_storage)
         : Namespace(namespace_iri, node_storage) {
         for (auto const &suffix : all_suffixes)
-            this->cache_.template emplace(suffix, IRI{namespace_iri_ + std::string{suffix}, node_storage}.backend_handle());
+            this->cache_.emplace(suffix, IRI{namespace_iri_ + std::string{suffix}, node_storage}.backend_handle());
```

Comment:
Is this change because of a warning. The template seemed fine to me.

Reply:
yes, this was fine just saw that by chance. Having `.template` is only necessary if you have 2 or more template arguments after the function name and this has none.